File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2020w/ID07IO1AB/Strategy/src/Sportsman.java
public class Sportsman
{
   private final String name;
   private final SportsmenStrategySwim swim;
   public Sportsman(String name,SportsmenStrategySwim swim)
   {
      this.name=name;
      this.swim=swim;
   }
   public String run() { return "Running"; }
   public String swim() { return swim.doit(); }
   public String jump() { return "Jumping"; }
}

File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2021s/Id06IO2/Command/Command/Sportsman.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    class Sportsman
    {
        private string name;
        public Sportsman(string name) { this.name=name; }
        public string Run() { return "Running"; }
        public string Jump() { return "Jumping"; }
        public string Swim() { return "Swimming"; }
        public string execute(SportsmenCommand cmd) { return name+": "+cmd.execute(this); }
    }
}
